### ISSUE TITLE
align podman experience with other runners

### ISSUE THREAD

**Issue by tomeichlersmith** [tomeichlersmith/denv#9]
need to get idmap support: https://github.com/containers/podman/commit/82a050a58f2ef2cf940c2ef49e1f8c4b17ab8beb

already merged into podman main, just waiting on release

**Comment by tomeichlersmith** [comment]
[This section of the root-less podman setup documentation](https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md#using-volumes) points me towards another running flag `--userns=keep-id` which is probably what we actually just want instead of the more advanced idmap thing
